packages/insomnia-app/app/ui/components/codemirror/extensions/autocomplete.ts

```diff
@@ -419,7 +419,7 @@ function matchSegments(listOfThings, segment, type, limit = -1) {
       // @ts-expect-error -- TSCONVERSION
       segment,
       // @ts-expect-error -- TSCONVERSION
-      displayValue,
+      displayValue: displayValue || '',
```

Comment:
if there's a display value that happens to be falsey like `0` this will return `''` when we might actually want it to show `0`.  I suggest we use `??` instead.  Thoughts?

```suggestion
      displayValue: displayValue ?? '',
```

Reply:
you caught me out, not-believing in zeros again